.github/CODEOWNERS

```diff
@@ -1,10 +1,12 @@
 l2geth/                             @smartcontracts @tynes @karlfloersch
+packages/specs/l2geth/              @smartcontracts @tynes @karlfloersch
+packages/contracts/                 @smartcontracts @ben-chain @maurelian
+packages/specs/protocol/            @smartcontracts @ben-chain @maurelian
```

Comment:
I moved `packages/contracts/` up to make the relationship between the specs and the implementation locations more obvious.